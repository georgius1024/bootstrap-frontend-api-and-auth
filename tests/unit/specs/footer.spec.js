import {
  shallowMount
} from '@vue/test-utils'

import Footer from '@/components/app/footer'

describe('footer.vue', () => {
  it('should render correct contents', () => {
    const wrapper = shallowMount(Footer, {
      propsData: {
        location: 'LOCATION-NAME',
        userName: 'USER-NAME'
      }
    })
    expect(wrapper.html()).toContain('LOCATION-NAME')
    expect(wrapper.html()).toContain('USER-NAME')
    expect(wrapper.element).toMatchSnapshot()
  })
})
