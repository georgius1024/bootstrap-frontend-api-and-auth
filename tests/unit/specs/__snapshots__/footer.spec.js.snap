// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer.vue should render correct contents 1`] = `
<vfooter-stub
  app=""
  dark=""
  height="32"
>
  <div
    class="mx-4"
  >
    LOCATION-NAME
  </div>
   
  <vspacer-stub />
   
  <div
    class="mx-4"
  >
    USER-NAME
  </div>
</vfooter-stub>
`;
